uefi/src/helpers/aligned_buffer.rs

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use alloc::alloc::{alloc, dealloc, Layout, LayoutError};
+
+/// Helper class to maintain the lifetime of a memory region allocated with a non-standard alignment.
+/// Facilitates RAII to properly deallocate when lifetime of the object ends.
```

Comment:
nit: 

"This uses the global Rust allocator under the hood, and thus not necessarily the."

Reply:
I assume you want me to add this line to the comments (?), but I don't know how the sentence should end :sweat_smile:.